src/Files.App/Actions/Global/ToggleFullScreenAction.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Files.App.Commands;
+using Files.App.Extensions;
+using Microsoft.UI.Windowing;
+using System.Threading.Tasks;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class ToggleFullScreenAction : IToggleAction
+	{
+		public string Label => "FullScreen".GetLocalizedResource();
```

Comment:
```suggestion
		public string Label { get; } = "FullScreen".GetLocalizedResource();
```

`=>` **is not the same as** `{ get; }`!!!
In this case using `=>` will execute `GetLocalizedResource()` *every time* the property is accessed, whereas using `{ get; }` will make it readonly and assign it *only once*

Reply:
It's done.